app/views/applications/show.html.erb

```diff
@@ -3,11 +3,10 @@
 <p>Address: <%= @application.full_address %></p>
 <p>
   Description: <%= @application.description %>
-  <% if @application.description == nil %>
+  <% if @application.description == nil && @application.has_pets? %>
```

Comment:
We should probably double check if we can use If statements in our views at our project check in.